apps/etterlatte-beregning/src/main/kotlin/beregning/BeregningRepository.kt

```diff
@@ -94,7 +92,12 @@ private fun toBeregningsperiode(row: Row): BeregningsperiodeDAO = with(row) {
         grunnlagMetadata = Metadata(
             sakId = long(BeregningsperiodeDatabaseColumns.SakId.navn),
             versjon = long(BeregningsperiodeDatabaseColumns.GrunnlagVersjon.navn)
-        )
+        ),
+        trygdetid = int(BeregningsperiodeDatabaseColumns.Trygdetid.navn),
+        regelResultat = stringOrNull(BeregningsperiodeDatabaseColumns.RegelResultat.navn)?.let {
+            objectMapper.readValue(it)
```

Comment:
Blir dette riktig da, hvorfor ikke `objectMapper.readTree(it)` ?

Reply:
Det der krever at `regelResultat` er `JsonNode` - mens `readValue(it)` aksepterer `ObjectNode`. Sjekker opp litt her.

Reply:
Det stemmer, burde fått chaina de kommentarene sammen :D